src/main/java/windycall/Deadline.java

```diff
@@ -0,0 +1,65 @@
+package windycall;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Deadline extends Task{
+
+    private LocalDate deadline;
+
+
+    public Deadline(String description, boolean status, String deadline) {
+        super(description, status);
+        this.deadline = processDateTime(deadline);
+    }
```

Comment:
Can we move the `processDateTime` to `Parser`? In my opinion, it would be better to pass `deadline` in as `LocalDate` and let `Parser` handles the `processDateTime` as this will be more fitting with the Single Responsibility Principle.

Reply:
Nice suggestion and it makes much sense!!